assets/javascript/app-no.js

```diff
@@ -0,0 +1,102 @@
+$("#start").on("click", function() {;
+  $("#start").remove();
+for(var i=0; i< questions.length); i++) {
+  $("#question-box").append("<h3"> + questions[i].question + "</h3>");
+for(var j=0; j<questions[i].answers.length; j++){
+  $("#question-box").append("<input type= "radio" name="question - +i+"value=")
+}
+}
+
+})
+
+document.getElementById("start").innerHTML = 003 + ":" + 20;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The numeric literal '003' will have at different value at runtime.](https://app.codacy.com/app/columbia-fsf-f2019/trivia-game/pullRequest?prid=4555492)